include/actuator/ActuatorFunctorsSimple.h

```diff
@@ -0,0 +1,190 @@
+// Copyright 2017 National Technology & Engineering Solutions of Sandia, LLC
+// (NTESS), National Renewable Energy Laboratory, University of Texas Austin,
+// Northwest Research Associates. Under the terms of Contract DE-NA0003525
+// with NTESS, the U.S. Government retains certain rights in this software.
+//
+// This software is released under the BSD 3-clause license. See LICENSE file
+// for more details.
+//
+
+#ifndef ACTUATORFUNCTORSSIMPLE_H_
+#define ACTUATORFUNCTORSSIMPLE_H_
+
+#include <actuator/ActuatorTypes.h>
+#include <actuator/ActuatorBulkSimple.h>
+#include <actuator/ActuatorFunctors.h>
+#include <NaluParsedTypes.h>
+#include <NaluEnv.h>
+
+namespace sierra {
+namespace nalu {
+
+  struct InterpActuatorDensity
+  {
+    using execution_space = ActuatorFixedExecutionSpace;
+
+    InterpActuatorDensity(ActuatorBulk& actBulk, stk::mesh::BulkData& stkBulk);
+
+    void operator()(int index) const;
+
+    ActuatorBulk& actBulk_;
+    stk::mesh::BulkData& stkBulk_;
+    VectorFieldType* coordinates_;
+    ScalarFieldType* density_;
+  };
+
+  // Things to calculate lift, drag, and AOA based on 2D airfoil theory
+  namespace AirfoilTheory2D {
+  void calculate_alpha(
+    Coordinates ws, 
+    Coordinates zeroalphadir, 
+    Coordinates spandir,
+    Coordinates chordnormaldir, 
+    double twist, 
+    Coordinates &ws2D, 
+    double &alpha);
+
+  void calculate_cl_cd(
+    double alpha,
+    std::vector<double> aoatable,
```

Comment:
Convert to Kokkos::View's

Reply:
I converted as many `std::vectors` to doubles as I could.